pkg/logging/config.go

```diff
@@ -24,7 +24,7 @@ import (
 )
 
 const (
-	ConfigName = "config-logging"
+	ConfigName = "serving-config-logging"
```

Comment:
Rather than changing it from one hard-coded name to another hard-coded name I would rather see the ConfigName taken dynamically like you did for namespace. This provides more flexibility for sharing (or not sharing) logging configuration across components.

Reply:
I agree with you. I update the code. @dgerd